packages/apps/transfer/.env.example

```diff
@@ -0,0 +1,10 @@
+NODE_ENV=development
+
+# Kadena Networks
+KADENA_API_TTIL=60000
```

Comment:
A TTL for a transaction of 60 seconds is fairly short (I don't know whether this is ms or seconds). Also good to mention what this default is used for
Depending on the use case different TTLs apply. For a transaction I'd say at least 3 but maybe more block heights, (3 being 90 sec).

Reply:
IMHO `60000*60` would be a nice default.

Reply:
changed it to 10 minutes